src/coreclr/jit/gentree.cpp

```diff
@@ -15378,34 +15378,18 @@ GenTree* Compiler::gtNewTempAssign(
         {
             ok = true;
         }
-        // 2) TYP_DOUBLE = TYP_FLOAT or TYP_FLOAT = TYP_DOUBLE
```

Comment:
Do you need to renumber in the comments now you're removed 2?

Reply:
Yep, thank you for noticing that.